sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,87 @@
+(28912,9,0,'Your High Inquisitor is nothing more than a pile of meat, Crusaders! There are none beyond the grasp of the Scourge!',14,0,100,22,0,0,29241,0,'Koltira Deathweaver');
+
+DELETE FROM `creature_summon_groups` WHERE `summonerId`=28912;
+INSERT INTO `creature_summon_groups` (`summonerId`,`summonerType`,`groupId`,`entry`,`position_x`,`position_y`,`position_z`,`orientation`,`summonType`,`summonTime`) VALUES
+(28912,0,0,29007,1640.672,-6032.053,134.8221,4.679538,6,5000),
+(28912,0,0,29007,1641.005,-6031.893,134.8221,0.4014257,6,5000),
+(28912,0,0,29007,1639.705,-6031.737,134.8221,2.443461,6,5000),
+(28912,0,1,29007,1640.404,-6032.289,134.7387,4.517092,6,5000),
+(28912,0,1,29007,1641.73,-6030.751,134.8221,6.143559,6,5000),
+(28912,0,1,29007,1639.466,-6030.404,134.8221,4.502949,6,5000),
+(28912,0,2,29007,1640.286,-6033.075,134.6049,4.679538,6,5000),
+(28912,0,2,29007,1642.04,-6030.384,134.8221,1.37881,6,5000),
+(28912,0,2,29007,1640.116,-6029.782,134.8221,5.707227,6,5000),
+(28912,0,2,29007,1640.995,-6029.803,134.8221,1.605703,6,5000),
+(28912,0,3,29001,1640.766,-6032.984,134.5805,4.596632,5,0);
+
+DELETE FROM `waypoint_data` WHERE `id`=28912;
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `delay`, `action`, `orientation`, `action_chance`, `move_type`) VALUES
+(28912,1,1640.65,-6032.49,134.739,0,0,0,100,1),
+(28912,2,1639.1,-6028.1,134.739,0,0,0,100,1),
+(28912,3,1628.37,-6026.34,134.739,0,0,0,100,1),
+(28912,4,1625.79,-6021.81,134.739,0,0,0,100,1),
+(28912,5,1627.06,-6016.95,134.739,0,0,0,100,1),
+(28912,6,1639.85,-6015.64,134.74,0,0,0,100,1),
+(28912,7,1653.376,-5971.874,132.2567,0,0,0,100,1),
+(28912,8,1653.61,-5927.44,121.345,0,0,0,100,1),
+(28912,9,1656.07,-5905.5,116.21,0,0,0,100,1),
+(28912,10,1681.736,-5888.621,116.257,0,1188,0,100,1);
+UPDATE `creature_addon` SET `bytes1`=0,`auras`='' WHERE `guid`=130354;
+
+-- High Inquisitor Valroth
+DELETE FROM `creature_text` WHERE `CreatureID`=29011;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(29011,0,0,'Acolytes, chain them all up! Prepare them for questioning!',14,0,100,1,0,0,29202,0,'High Inquisitor Valroth'),
+(29011,1,0,'Scourge filth! By the Light be cleansed!',14,0,100,0,0,0,29214,0,'High Inquisitor Valroth'),
+(29011,2,0,'The Crusade will purge your kind from this world!',14,0,100,0,0,0,29215,0,'High Inquisitor Valroth'),
+(29011,3,0,'It seems that I''ll need to deal with you myself. The High Inquisitor comes for you, Scourge!',14,0,100,0,0,0,29216,0,'High Inquisitor Valroth');
+
+DELETE FROM `creature_text` WHERE `CreatureID`=29001;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `ID`, `Text`, `Type`, `Language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(29001,0,0,'Come out and play.',12,0,100,0,0,0,29219,0,'High Inquisitor Valroth'),
```

Comment:
The text here is wrong, it must be "Coward!"

Reply:
https://wow.gamepedia.com/Bloody_Breakout#Notes